x-pack/plugins/triggers_actions_ui/public/application/sections/field_browser/field_items.tsx

```diff
@@ -118,11 +102,11 @@ const getDefaultFieldTableColumns = (highlight: string): FieldTableColumns => [
           <EuiScreenReaderOnly data-test-subj="descriptionForScreenReaderOnly">
             <p>{i18n.DESCRIPTION_FOR_FIELD(name)}</p>
           </EuiScreenReaderOnly>
-          <TruncatableText>
-            <Description data-test-subj={`field-${name}-description`}>
+          <span css={styles.truncatable}>
+            <span css={styles.description} data-test-subj={`field-${name}-description`}>
               {`${description ?? getEmptyValue()} ${getExampleText(example)}`}
```

Comment:
Yeap, it would make sense, but we are inside the column `render` function here, it is not a component or a hook, so we can't call `useMemo`, we would need to extract the column rendering into components to do so.